src/lib.rs

```diff
@@ -3,8 +3,11 @@
 
 include!(concat!(env!("OUT_DIR"), "/helium.rs"));
 
+mod types;
+
 pub use blockchain_txn::Txn;
 pub use prost::{DecodeError, EncodeError, Message};
+pub use types::*;
```

Comment:
can we just `pub mod types`. I've come to like the name spacing instead of exporting everything at the top level. 

The concern is that the "models" in types are not proto messages so they shouldn't live in the same namespace.

Reply:
updated [a45d99](https://github.com/helium/proto/pull/334/commits/9a45d99267e757def7782d339f7c63e78d2193bd)